packages/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,60 @@
+import { forwardRef } from "react";
+
+import {
+  Arrow,
+  Content,
+  Provider,
+  Root,
+  Trigger,
+} from "@radix-ui/react-tooltip";
+import { twMerge } from "tailwind-merge";
+
+export type TooltipProviderProps = {
+  children: React.ReactNode;
+  /**
+   * The duration from when the pointer enters the trigger until the tooltip gets opened.
+   * @defaultValue 700
+   */
+  delayDuration?: number;
+  /**
+   * How much time a user has to enter another trigger without incurring a delay again.
+   * @defaultValue 300
+   */
+  skipDelayDuration?: number;
+  /**
+   * When `true`, trying to hover the content will result in the tooltip closing as the pointer leaves the trigger.
+   * @defaultValue false
+   */
+  disableHoverableContent?: boolean;
+};
+const TooltipProvider: React.FC<TooltipProviderProps> = Provider;
+
+const Tooltip = Root;
+
+const TooltipArrow = Arrow;
+
+const TooltipTrigger = Trigger;
+
+const TooltipContent = forwardRef<
+  React.ElementRef<typeof Content>,
+  React.ComponentPropsWithoutRef<typeof Content>
+>(({ className, sideOffset = 4, ...props }, ref) => (
+  <Content
+    ref={ref}
+    sideOffset={sideOffset}
+    className={twMerge(
+      "z-50 overflow-hidden rounded-md border bg-popover px-3 py-1.5 text-sm text-popover-foreground shadow-md animate-in fade-in-0 zoom-in-95 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2",
```

Comment:
Let's update this styles to match the design. This should work on dark and light mode too. Most styles are ok but colors, radius, shadow don't look ok.

Reply:
You're right! I had styles colliding between this tooltip and the content child component.